### ISSUE TITLE
Website does not respect explicit language choice

### ISSUE THREAD

**Issue by Schweinepriester** [bitwarden/clients#8507]
### Steps To Reproduce

TL;DR: Please make vault.bitwarden.com respect the selected language in settings, at least when logged in.

1. Go to https://vault.bitwarden.com/ with Browser/OS not set to desired language of Bitwarden, in my case DE, desired EN
2. See everything is in DE, presumably because of autodetect/`Accept-Language` header
3. Log in, go to https://vault.bitwarden.com/#/settings/preferences, change language to EN, all good
4. Close the tab
5. Go to https://vault.bitwarden.com/ again, everything is in DE again, even though my email address is remembered as well
    - from my perspective it's debatable which language should be used here, but I would desire EN, seeing that my login is remembered
6. Log in again, still everything in DE, despite changing it before

### Expected Result

Website remembers explicit language choice from the settings.
(aka. language setting is account bound?)

### Actual Result

Website changes language back based on some form of autodetection.

### Operating System

Windows, macOS

### Web Browser

Chrome

### Build Version

Version 2024.3.0

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by NovaSilentium** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!
